lib/src/dom/async/wait_for.dart

```diff
@@ -168,7 +168,7 @@ Future<T> waitFor<T>(
 Future<void> waitForElementToBeRemoved(
   Node Function() callback, {
   Node container,
-  Duration timeout,
+  Duration timeout = const Duration(seconds: 1),
```

Comment:
Would it make more sense to default to the timeout set in the config like `waitFor` does [here](https://github.com/Workiva/react_testing_library/blob/53492c345eac55154e816fff949c7f01ee19259d/lib/src/dom/async/wait_for.dart#L67)?

Reply:
Yep, I think it would. I'll make that change.

Related—are we committed to the 1 second timeout? How about we set this timeout to `config.asyncUtilTimeout` as well?

Reply:
Yes that's what I meant use `config.asyncUtilTimeout` instead of 1 second here.

Reply:
Heh sorry, I gotcha. Is there any functional need to move the `timeout` assignment out and use the null assignment operator like that same line does?

```
timeout ??= ...
```

Reply:
No worries! Oh I think you would have to move it out and use the null assignment just because you'd need the `final config = getConfig();` line to access the `config`

Reply:
Heh I gotcha, sorry bout that. Updated